cmd/rr-temporal.go

```diff
@@ -1,12 +1,58 @@
 package main
 
+import (
+	"github.com/spiral/endure"
+	"github.com/spiral/roadrunner/v2/plugins/config"
+	"github.com/spiral/roadrunner/v2/plugins/factory"
+	"github.com/temporalio/roadrunner-temporal/cmd/subcommands"
+	"github.com/temporalio/roadrunner-temporal/plugins/temporal"
+)
+
 func main() {
 	/*
-	1. -c flag - path to the config (default .rr.yaml
-	2. change working dir to the path with .rr.yaml
-	3. init endure here, path should be -c
-	4. command to serve endure container
-	5. plugins to use - temporal, config, factory
-	6. USE COBRA!!!!!11111oneoneone
+		1. -c flag - path to the config (default .rr.yaml
+		2. change working dir to the path with .rr.yaml
+		3. init endure here, path should be -c
+		4. command to serve endure container
+		5. plugins to use - temporal, config, factory
+		6. USE COBRA!!!!!11111oneoneone
 	*/
+
+	var err error
+	subcommands.Container, err = endure.NewContainer(endure.DebugLevel, endure.RetryOnFail(false))
+	if err != nil {
+		panic(err)
+	}
+
+	err = subcommands.Container.Register(&temporal.Plugin{})
+	if err != nil {
+		panic(err)
+	}
+
+	conf := &config.ViperProvider{}
+	conf.Path = subcommands.ConfDir
+	conf.Prefix = "rr"
+
+	err = subcommands.Container.Register(conf)
+	if err != nil {
```

Comment:
can we use something like "Container.MustRegister", it's too verbose

Reply:
What's the difference between Register and MustRegister in terms of behavior?

Reply:
mustRegister triggers panic on error

Reply:
You can trigger panic on the error which the Register method returns if you wish. What is the point of hidden panic? This is not really obvious, because you should otherwise have a panic handler to free allocated resources. Also, this is not a case for Endure which handles internal panics as much as possible to return to a user nice errors.

Reply:
Panics here are just temporary. It will be replaced with the logger and return in the scope of another ticket.

Reply:
Done. Panics removed from main, replaced with proper error handling via logger